smart-notes/rag_mvp/pipelines/embedding_pipeline.py

```
@@ -0,0 +1,47 @@
+        return embeddings
+
+    def process_notes(self, text):
+        self.chunks = self.chunk_text(text)
+        embeddings = self.build_index(self.chunks)
+        return self.chunks, embeddings
+
+    def semantic_search(self, query, top_k=3):
+        query_vec = self.model.encode([query])
+        query_vec = np.array(query_vec).astype("float32")
+
+        distances, indices = self.index.search(query_vec, top_k)
```

Ruff (0.15.0)

[warning] Unpacked variable `distances` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)